mscore/musescore.cpp

```diff
@@ -3210,39 +3213,79 @@ static bool doConvert(Score* cs, QString fn, QString plugin = "")
                   cs->setLayoutMode(LayoutMode::PAGE);
                   cs->doLayout();
                   }
-            if (mscore->loadPlugin(plugin))
+            if (mscore->loadPlugin(plugin)) {
+                  fprintf(stderr, "\tusing plugin <%s>\n", qPrintable(plugin));
                   mscore->pluginTriggered(0);
+                  }
             mscore->unloadPlugins();
             if (layoutMode1 != cs->layoutMode()) {
                   cs->setLayoutMode(layoutMode1);
                   cs->doLayout();
                   }
             }
+      foreach (const QJsonValue& outFile, outFiles) {
```

Comment:
better use `for (const QJsonValue& outFile : outFiles) {`

Reply:
Thanks, fixed in the next push.